.gitignore

```diff
@@ -12,3 +12,30 @@ components/*/.output/
 .cproject
 .project
 .settings/
+.vscode/settings.json
```

Comment:
Can you make this a `.vscode/**` please.

Reply:
Fixed.